.github/workflows/main.yml

```diff
@@ -108,18 +111,29 @@ jobs:
           cd .. && mv quay-operator/bundle/manifests operators/project-quay/${{ github.event.inputs.tag }} && \
           rm -rf quay-operator operators/project-quay/${{ github.event.inputs.tag }}/quay-operator.service.yaml
 
+      - name: Get and Set Old Version
+        run: echo "OLD_VERSION=$(yq -P '.channels[] | select(.name == "${{ github.event.inputs.channel" }}) .currentCSV' operators/project-quay/project-quay.package.yaml)" >> $GITHUB_ENV
```

Comment:
`"${{ github.event.inputs.channel" }}` needs to be replaced by "${{ github.event.inputs.channel }}"

Reply:
Should be fixed now